dotnet/src/AutoGen.Core/Orchestrator/RolePlayOrchestrator.cs

```diff
@@ -80,23 +90,101 @@ public RolePlayOrchestrator(IAgent admin, Graph? workflow = null)
             options: new GenerateReplyOptions
             {
                 Temperature = 0,
-                MaxToken = 128,
-                StopSequence = [":"],
+                MaxToken = 1024,
+                StopSequence = ["//finish"],
```

Comment:
Maybe use `}}` (The close tag for JSON output) instead of `//finish` as stop word?

Reply:
the output is like:
{
Speaker:"xxx",
Message:"xxx"
}//finish

In order to prevent any additional instructions from being appended to the model.

Reply:
I'm currently unable to determine whether the Agent Name will contain any special characters set by developers, so these symbols are not used.